src/libserver/re_cache.c

```diff
@@ -481,7 +480,6 @@ rspamd_re_cache_init (struct rspamd_re_cache *cache, struct rspamd_config *cfg)
 	const gchar *platform = "generic";
 	rspamd_fstring_t *features = rspamd_fstring_new ();
 
-	cache->disable_hyperscan = cfg->disable_hyperscan;
```

Comment:
This should not be removed - it is an used feature to disable HS from the configuration.